opal/mca/common/ofi/common_ofi.h

```diff
@@ -38,6 +38,7 @@ OPAL_DECLSPEC void opal_common_ofi_mca_register(void);
 OPAL_DECLSPEC void opal_common_ofi_mca_deregister(void);
 OPAL_DECLSPEC struct fi_info* opal_common_ofi_select_ofi_provider(struct fi_info *providers, 
                                                                   char *framework_name);
+OPAL_DECLSPEC int opal_common_ofi_is_in_list(char **list, char *item);
```

Comment:
i'd love some documentation :).

Reply:
I tried my best, it's kind of a funky function. Maybe the provider list logic that currently exists should be changed to explicitly call out layered providers (ie. tcp, tcp;ofi_rxm) as separate providers.